web/src/test/java/org/cbioportal/web/util/DataBinnerTest.java

```diff
@@ -686,13 +689,19 @@ public void testStaticDataBinnerFilter() {
 
         List<ClinicalData> unfilteredClinicalData = mockClinicalData(attributeId, studyId, values);
         List<String> unfilteredPatientIds =
-            unfilteredClinicalData.stream().map(ClinicalData::getPatientId).collect(Collectors.toList());
+            unfilteredClinicalData
+            .stream()
+            .map(datum-> studyViewFilterUtil.getCaseUniqueKey(datum.getStudyId(), datum.getPatientId()))
+            .collect(Collectors.toList());
```

Comment:
since now this became a rather complex multiple line filtering logic, instead of copying and pasting it everywhere should we create a method for it?